sphinx/util/compat.py

```diff
@@ -10,31 +10,7 @@
 """
 from __future__ import absolute_import
 
-import warnings
-
-from docutils import nodes
 from docutils.parsers.rst import Directive  # noqa
 
 from docutils import __version__ as _du_version
 docutils_version = tuple(int(x) for x in _du_version.split('.')[:2])
-
-
-def make_admonition(node_class, name, arguments, options, content, lineno,
```

Comment:
Removing descriptions is ok to me. 'deprecation' is indicated, so there is nothing to worry about.